src/server/ocpp/validation/OCPPValidation.ts

```diff
@@ -108,6 +109,8 @@ export default class OCPPValidation extends SchemaValidator {
   public async validateMeterValues(tenantID: string, chargingStation: ChargingStation, meterValues: OCPPMeterValuesRequestExtended): Promise<void> {
     // Always integer
     meterValues.connectorId = Utils.convertToInt(meterValues.connectorId);
```

Comment:
Utils.convertToInt() is not need, the schema should convert it already.
You can set the default value to 0 in the schema for the connectorId.